repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Repeater.scala

```diff
@@ -62,10 +62,8 @@ object Repeater extends SafeIOApp {
           case Left(NonFatal(e)) =>
             System.out.println(e.toString)
             IO.raiseError(e) >> IO.pure(ExitCode.Error)
-          case Left(e) =>
-            IO(System.err.println(s"Snowplow BigQuery Repeater failed to start. ${e.toString}")) >> IO
-              .pure(ExitCode.Error)
```

Comment:
I'm slightly confused now, because I would expect the compiler to complain about non-exhaustive pattern matching.  By matching just `Left(NonFatal(e))` it means nothing is matching `Left(e)` where e _is_ fatal.  I checked the compiler but there are no warnings.  To be clear, I don't want us to catch and handle fatal warnings.  But pattern matching in a `flatMap` is a different case, because the exception has already been caught and wrapped into an Either.

I don't think we need to investigate this further though.  If the compiler is happy then I am happy.

Reply:
The way I understand it, we want to allow fatal errors to stop the program; and if `flatMap` sees an unmatched case it will throw and exception which will have the same effect.